JPS_Ontology/ontology/ontobuiltenv/OntoBuildingEnvironment.csv

```diff
@@ -118,3 +118,10 @@ representativeFor,Object Property,,,AveragePricePerSqm,PostalCode,,A relationshi
 representativeFor,Object Property,,,AveragePricePerSqm,PostalCode,exactly 1,,https://www.theworldavatar.com/kg/ontobuiltenv/,representative for
 Bank,Class,Non-Domestic,IS-A,,,,A property where customers can deposit and withdraw money.,https://www.theworldavatar.com/kg/ontobuiltenv/,Bank
 Pharmacy,Class,Medical,IS-A,,,,A healthcare facility where medicinal drugs are prepared or sold.,https://www.theworldavatar.com/kg/ontobuiltenv/,Pharmacy
+BuildingFloors,Class,,,Property,,,A concept to describe amount of floors in a building.,https://www.theworldavatar.com/kg/ontobuiltenv/,Building floors
+hasBuildingFloors,Object Property,,,Property,BuildingFloors,exactly 1,A relationship between a property and its amount floors.,https://www.theworldavatar.com/kg/ontobuiltenv/,has building floors
+hasFloorsValue,Data Property,,,BuildingFloors,integer,exactly 1,A relationship between a building floors and its value.,https://www.theworldavatar.com/kg/ontobuiltenv/,has floors value
+hasAccuracy,Data Property,,,BuildingFloors,string,exactly 1,A relationship between a building floors and its accuracy level.,https://www.theworldavatar.com/kg/ontobuiltenv/,has accuracy
+hasGFA,Data Property,,,Property,float,exactly 1,A relationship between a building and its GFA value.,https://www.theworldavatar.com/kg/ontobuiltenv/,has GFA
+hasCountry,Data Property,,,http://ontology.eil.utoronto.ca/icontact.owl#Address,string,exactly 1,The country where the address is located.,https://www.theworldavatar.com/kg/ontobuiltenv/,has country
```

Comment:
Might be good to keep consistency with previous way of writing the relationship in address 

Example: 
A relationship between an address and the respective country.
[hasStreet](https://github.com/cambridge-cares/TheWorldAvatar/blob/45776dd293a74ac55b8a3ff679cfaa37c43aa2e2/JPS_Ontology/ontology/ontobuiltenv/OntoBuildingEnvironment.csv#L27)

Reply:
Was following how the comments were labelled inside the icontact ontology, where the Address and has... concepts originally came from. Since we are extending inside OntoBuiltEnv instead of icontact, I guess it would be better to follow the convention inside OntoBuiltEnv instead of icontact. Will update accordingly.